Mathlib/Analysis/Normed/Field/Basic.lean

```diff
@@ -199,18 +199,38 @@ variable (α) [NontriviallyNormedField α]
 theorem exists_one_lt_norm : ∃ x : α, 1 < ‖x‖ :=
   ‹NontriviallyNormedField α›.non_trivial
 
+theorem exists_one_lt_nnnorm : ∃ x : α, 1 < ‖x‖₊ := exists_one_lt_norm α
+
+theorem exists_one_lt_enorm : ∃ x : α, 1 < ‖x‖ₑ :=
+  exists_one_lt_nnnorm α |>.imp fun _ => ENNReal.coe_lt_coe.mpr
+
 theorem exists_lt_norm (r : ℝ) : ∃ x : α, r < ‖x‖ :=
   let ⟨w, hw⟩ := exists_one_lt_norm α
   let ⟨n, hn⟩ := pow_unbounded_of_one_lt r hw
   ⟨w ^ n, by rwa [norm_pow]⟩
 
+theorem exists_lt_nnnorm (r : ℝ≥0) : ∃ x : α, r < ‖x‖₊ := exists_lt_norm α r
+
```

Comment:
Can you also add an enorm version? Thanks!

Reply:
What would the statement be?

Reply:
This one?
```suggestion
theorem exists_lt_enorm (r : ℝ≥0) : ∃ x : α, r < ‖x‖ₑ := by
  obtain ⟨x, hx⟩ := exists_lt_nnnorm α r
  exact ⟨x, by simp [hx]⟩

Reply:
I'd argue that this one is less useful than the others, since the hypothesis is not of type `ENNReal`, and the RHS arguably should be norm-casted to the `nnnorm` version.

Reply:
@grunweg, do you agree with the above?

Reply:
You can do `{r : ENNReal} (hr : r \ne \top)` instead.

Reply:
Done

Reply:
The new version looks good to me, thanks!